base/replutil.jl

```diff
@@ -444,6 +456,10 @@ function show_method_candidates(io::IO, ex::MethodError, kwargs::Vector=Any[])
             buf = IOBuffer()
             tv = Any[]
             sig0 = method.sig
+            # @Hardcoded: don't display the sysimg.jl method
```

Comment:
What we have used before is TODO or FIX, etc. So if there's need for this, HARCODED would at least be consistent in style.